ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/OngoingStreamAuthIT.scala

```diff
@@ -86,25 +86,22 @@ final class OngoingStreamAuthIT
       submitAndWaitF = () =>
         submitAndWait(token = tokenAlice, party = partyAlice, applicationId = applicationId)
       _ <- submitAndWaitF()
-      streamObserver = observeTransactionsStream(tokenAlice, partyAlice)
+      _ = observeTransactionsStream(tokenAlice, partyAlice)
       _ <- submitAndWaitF()
       // Making a change to the user Alice
       _ <- grantUserRightsByAdmin(
         userId = userIdAlice,
         Right(Right.Kind.CanActAs(Right.CanActAs(UUID.randomUUID().toString))),
       )
-      _ = Thread.sleep(UserManagementCacheExpiryInSeconds.toLong * 1000)
-      //
-      // Timer that finishes the stream in case it isn't aborted as expected
-      timerTask = new TimerTask {
-        override def run(): Unit = streamObserver.onError(
-          new AssertionError("Timed-out waiting while waiting for stream to abort")
+      _ <- Delayed.Future.by(2.second)(
```

Comment:
Why not make it relative to `UserManagementCacheExpiryInSeconds` size i.e. either
`UserManagementCacheExpiryInSeconds*2` or `UserManagementCacheExpiryInSeconds+1`?

Reply:
Changed to `UserManagementCacheExpiryInSeconds + 1`